index.html

```diff
@@ -37,7 +56,6 @@
     <div style="margin: 0px; padding: 0px; border: 0px solid rgb(6, 117, 211); font-size: 19px; font-style: inherit; font-variant: inherit; font-weight: inherit; font-stretch: inherit; line-height: 1; font-family: sans-serif; vertical-align: baseline; box-sizing: content-box; overflow: unset; height: 1px; position: relative; width: auto; display: flex; background-color: rgb(9, 150, 248); color: rgb(255, 255, 255); text-align: left; align-items: center; border-radius: 3px; justify-content: center; text-decoration: none;"></div>
   </body>
 </html>
-<script src="scripts/jquery.js"></script>
 <script src="scripts/popup.js"></script>
 <script src="scripts/bootstrap.min.js"></script>
```

Comment:
in console if you see bootstrap.min.js is showing error as "bootstrap's javascript needs jquery " and we do not want that as jquery can make extension slow so you may have to remove bootstrap's js